actionpack/lib/action_dispatch/http/upload.rb

```diff
@@ -41,8 +42,21 @@ def initialize(hash) # :nodoc:
           @original_filename = nil
         end
 
-        @content_type      = hash[:type]
-        @headers           = hash[:head]
+        if hash.key?(:type)
+          ActiveSupport::Deprecation.warn(<<-MSG.squish)
+            Initialising with key `type` is deprecated and will be removed in Rails 6.0.
+            Please use key `content_type` instead.
+          MSG
+        end
+        if hash.key?(:head)
+          ActiveSupport::Deprecation.warn(<<-MSG.squish)
+            Initialising with key `head` is deprecated and will be removed in Rails 6.0.
```

Comment:
Needs to be Rails 6.1 now. Same in the other one.

Reply:
This change would be released in 6.1, so the deprecation should mention 6.2, right?

Reply:
Hey, let me enjoy my off-by-one errors in peaceful ignorance, please 😂

You're right, it's 6.2, thanks for catching it! ❤️